test/Eryph.ConfigModel.Catlets.Tests/FodderGeneConfigJsonSerializerTests.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Text.Json;
+using CultureAwareTesting.xUnit;
+using Eryph.ConfigModel.Json;
+using FluentAssertions;
+
+namespace Eryph.ConfigModel.Catlet.Tests;
+
+public class FodderGeneConfigJsonSerializerTests : FodderGeneConfigSerializerTestBase
+{
+    private const string ComplexConfigJson =
+        """
+        {
+          "name": "fodder1",
+          "variables": [
+            {
+              "name": "first",
+              "value": "first value"
+            },
+            {
+              "name": "second",
+              "type": "Boolean",
+              "value": "true",
+              "secret": true,
+              "required": true
+            }
+          ],
+          "fodder": [
+            {
+              "name": "admin-windows",
+              "remove": true,
+              "type": "cloud-config",
+              "content": "users:\n  - name: Admin\ngroups: [ \u0022Administrators\u0022 ]\n  passwd: InitialPassw0rd",
+              "file_name": "filename",
```

Comment:
Please change property to filename for readability (its ok when property changes breaking on type)

Reply:
Fixed. I have changed `FodderConfig.FileName` to `FodderConfig.Filename`. This is what you meant?

Reply:
yes